templates/next/pages/foos/[id]/edit.tsx

```diff
@@ -2,15 +2,23 @@ import { NextComponentType, NextPageContext } from 'next';
 import { Form } from '../../../components/{{{lc}}}/Form';
 import { {{{ucf}}} } from '../../../types/{{{ucf}}}';
 import { fetch } from '../../../utils/dataAccess';
+import Head from "next/head";
 
 interface Props {
   {{{lc}}}: {{{ucf}}};
 };
 
 const Page: NextComponentType<NextPageContext, Props, Props> = ({ {{{lc}}} }) => {
-
   return (
-    <Form {{{lc}}}={ {{{lc}}} }/>
+    <div>
+      <div>
+        <Head>
+          <title>{ {{{lc}}} && `Edit {{{ucf}}} ${ {{~lc}}['@id']}`}</title>
+          <meta property="og:title" content={ {{{lc}}} && `Edit{{{ucf}}} ${ {{~lc}}['@id']}`} key="title" />
```

Comment:
Is this necessary if the content is the same as the `<title>` tag?

Reply:
It seems that the `<title>` tag is used by Google and `meta property="og:title"` is used by social medias.
(see: https://www.quora.com/In-terms-of-SEO-value-does-the-Open-Graph-meta-title-tag-meta-property-og-title-have-as-much-value-as-a-Title-tag-Can-we-leave-title-tag-out, it's old but maybe still valid, see NextJS documentation examples about `<Head />` component : https://nextjs.org/docs/api-reference/next/head)